conda.recipe/meta.yaml

```diff
@@ -20,6 +20,7 @@ requirements:
     - python
     - setuptools
     - pytz
+    - future
```

Comment:
Why are we mixing six and future ?

Seem like related libraries to accomplish something similar (albeit in a different way) Is this something we want to pursue?

Reply:
Fixed